app/controllers/application_controller.rb

```diff
@@ -283,4 +283,12 @@ def prepare_exception_notifier
       current_user_browser: browser.to_s,
     }
   end
+
+  # For Lograge
+  def append_info_to_payload(payload)
+    super
+    payload[:host] = request.host
+    payload[:remote_ip] = request.remote_ip
```

Comment:
This is already in the same file on line 133.  I think you could just add 
`payload[:ip]` up there.

Reply:
👍